app/src/sharedTest/java/org/oppia/app/topic/info/TopicInfoFragmentTest.kt

```diff
@@ -55,6 +56,7 @@ private const val DUMMY_TOPIC_DESCRIPTION_LONG =
 class TopicInfoFragmentTest {
   private val topicThumbnail = R.drawable.lesson_thumbnail_graphic_child_with_fractions_homework
   private val internalProfileId = 0
+  private val context = ApplicationProvider.getApplicationContext<Context>()
```

Comment:
Please remove all the changes that you are doing in this PR. These test cases will get updated in #1480

Reply:
Done